serf/keymanager.go

```diff
@@ -83,6 +87,12 @@ func (k *KeyManager) streamKeyResp(resp *KeyResponse, ch <-chan NodeResponse) {
 			}
 		}
 
+		if _, ok := resp.PrimaryKeys[nodeResponse.PrimaryKey]; !ok {
+			resp.PrimaryKeys[nodeResponse.PrimaryKey] = 1
+		} else {
+			resp.PrimaryKeys[nodeResponse.PrimaryKey]++
+		}
```

Comment:
```suggestion
		resp.PrimaryKeys[nodeResponse.PrimaryKey] += 1
```

This works because if the key doesn't exist the zero value is returned.